spec/fs_spec.lua

```diff
@@ -47,11 +47,665 @@ describe("Luarocks fs test #whitebox #w_fs", function()
          assert.same(false, fs.is_file("/nonexistent"))
       end)
 
-      it("returns false when arguments exists but is not a file", function()
+      it("returns false when the argument exists but is not a file", function()
          tmpdir = os.tmpname()
          os.remove(tmpdir)
          lfs.mkdir(tmpdir)
          assert.same(false, fs.is_file("/nonexistent"))
       end)
    end)
+
+   describe("fs.is_dir", function()
+      local tmpfile
+      local tmpdir
+
+      after_each(function()
+         if tmpfile then
+            os.remove(tmpfile)
+            tmpfile = nil
+         end
+         if tmpdir then
+            lfs.rmdir(tmpdir)
+            tmpdir = nil
+         end
+      end)
+
+      it("returns true when the argument is a directory", function()
+         tmpdir = os.tmpname()
+         os.remove(tmpdir)
```

Comment:
just a reminder that `os.tmpname` usually does not include a path on Windows, but on other platforms it does. 

So no biggy, but using it might leave artefacts in the test directory if those tests fail.

example: https://github.com/stevedonovan/Penlight/blob/2fb9b303f349955597c229f767d421423eaaf2c8/lua/pl/path.lua#L385-L396